steamapi/store.py

```diff
@@ -0,0 +1,53 @@
+__author__ = 'andrew'
+
+import uuid
+from .core import APIConnection
+
+
+class SteamStore(object):
+    def __init__(self, appid, debug=False):
+        self.appid = appid
+        self.interface = 'ISteamMicroTxnSandbox' if debug else 'ISteamMicroTxn'
+
+    def get_user_microtxh_info(self, steamid):
+        params = {
+            'steamid': steamid,
+            'appid': self.appid
+        }
+        return APIConnection().call(self.interface, 'GetUserInfo', 'v1', **params)
+
+    def init_purchase(self, steamid, itemid, amount, **kwargs):
+        params = {
+            'steamid': steamid,
+            'itemid[0]': itemid,
+            'amount[0]': amount,
+            'appid': self.appid,
+            'orderid': uuid.uuid1().int >> 64,
+            'itemcount': kwargs.get('itemcount', 1),
+            'language': kwargs.get('language', 'en'),
+            'currency': kwargs.get('currency', 'USD'),
+            'qty[0]': kwargs.get('qty', 1),
+            'description[0]': kwargs.get('description', 'Some description'),
```

Comment:
Use function defaults instead of this implementation. (This helps users who try out `steamapi` with a console, or use an IDE like PyCharm)